### ISSUE TITLE
tq claim triggers ConcurrentModificationException

### ISSUE THREAD

**Issue by bluelightning32** [TerminatorNL/Tiquality#29]
Modpack: Manufactio version 0.95
Forge: forge-1.12.2-14.23.5.2822-universal
Tiquality: Tiquality-FAT-1.12.2-GAMMA-1.4.1

Whenever a player runs `/tq claim`, the server crashes. When the server starts back up, the chunk is claimed. A player triggered this twice in a row on the server before I removed Tiquality again.

```
java.util.ConcurrentModificationException
        at java.util.HashMap$HashIterator.remove(HashMap.java:1459)
        at net.minecraft.world.gen.ChunkProviderServer.tick(ChunkProviderServer.java:262)
        at net.minecraft.world.WorldServer.tick(WorldServer.java:207)
        at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:756)
        at net.minecraft.server.dedicated.DedicatedServer.updateTimeLightAndEntities(DedicatedServer.java:397)
        at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668)
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
        at java.lang.Thread.run(Thread.java:748)
```

Crash report: https://gist.github.com/bluelightning32/746748c74b73e388aef46add939118e5

**Comment by TerminatorNL** [comment]
Thank you for the report, this is fixed in 1.4.2!

**Comment by TerminatorNL** [comment]
I'm going to re-write this piece of code later on, because the reworked storage solution I have made in version 1.4.0 now allows me to not have direct access to the chunks, meaning the server does not have to load them.
